mdapi/services/__init__.py

```diff
@@ -51,11 +51,13 @@ async def _get_package(brch, name=None, actn=None, srcn=None):
     pckg = None
     wrongdbs = False
 
+    print("IDIOTA", standard.DB_FOLDER)
```

Comment:
I think this print shouldn't be here

Reply:
It most definitely should not be haha. ;P

Reply:
Fixed with https://github.com/t0xic0der/mdapi/pull/36/commits/51d007e64185a5bf366e77b6f35569e003c37a3f